Assets/Scripts/Player/PlayerController.cs

```diff
@@ -72,6 +72,7 @@ public class PlayerController : MonoBehaviour {
 	public float JumpLength => jumpEndTime;
 	public float HumanJumpLength => humanJumpEndTime;
 	public float AirJumpLength => airJumpEndTime;
+	
```

Comment:
You added a newline to PlayerController?

Reply:
I don't think I've even opened this script? ... but otherwise, it must have been unintentional